src/components/ConnectionError/ConnectionError.test.tsx

```diff
@@ -38,4 +40,20 @@ describe("ConnectionError", () => {
     await userEvent.click(screen.getByRole("button", { name: "refreshing" }));
     expect(window.location.reload).toHaveBeenCalled();
   });
+
+  it("displays Audit Logs errors", async () => {
+    const state = rootStateFactory.withGeneralConfig().build({
+      juju: { auditEvents: { errors: "Oops!" } },
+    });
+    const {
+      result: { rerender },
+    } = renderComponent(<ConnectionError />, { state });
+    rerender(<Toaster />);
```

Comment:
I'm not sure how we can test this bit without a forced rerender, considering that the `Toast` is triggered inside a `useEffect`. Is there a better way to do this part?